OpenTabletDriver.UX/MainForm.cs

```diff
@@ -603,7 +603,8 @@ private async Task ExportDiagnostics()
                 {
                     case DialogResult.Ok:
                     case DialogResult.Yes:
-                        var file = new FileInfo(fileDialog.FileName);
+                        String[] options = { ".json", ".txt", ".log" };
```

Comment:
```suggestion
                        string[] options = { ".json", ".txt", ".log" };
```
Sorry, our example snippet made this "mistake" as well, but C# code style tends to prefer using the alias `string` over the class `String`. They both compile down to the same thing, so it's just cosmetic consistency. You can fix this and squash it once again. Looks good to merge otherwise.

Reply:
P.S., you could go a step further by making it a `const` variable at the top of the method.

Reply:
@vedattt 
Thank you for review.
I fixed it.